tests/integration/devfile/utils/utils.go

```diff
@@ -600,11 +600,6 @@ func validateContainerExecListDir(odoV1Watch OdoV1Watch, odoV2Watch OdoV2Watch,
 			dir := envs["ODO_S2I_SRC_BIN_PATH"]
 			stdOut = ocRunner.ExecListDir(podName, project, filepath.ToSlash(filepath.Join(dir, "src")))
 		}
-	case "docker":
```

Comment:
There is one more occurrence in this file here https://github.com/openshift/odo/pull/4521/files#diff-4b1a4ef4eda8c10bd4b6b741da30321a8d2e2a50a5640d00a7e10051ee31087dR685

Reply:
Raised a PR https://github.com/openshift/odo/pull/4524